namer/metadataapi.py

```diff
@@ -188,7 +188,7 @@ def __match_weight(result: ComparisonResult) -> float:
     value = 0.00
     if result.phash_distance is not None:
         logger.debug("Phash match with '{} - {} - {}'", result.looked_up.site, result.looked_up.date, result.looked_up.name)
-        value = 1000.00 - result.phash_distance * 100
+        value += 1000 - result.phash_distance * 125
         if result.site_match:
```

Comment:
maybe move these other if blocks out of the phash block and switch name match below to be (900 + the name percent) that way a phash with distance of 2 has a fighting chance to be first?

Reply:
just thinking outlaid about how to balance all the permutations.